mock.js

```diff
@@ -0,0 +1,4 @@
+import mockAsyncStorage from '@react-native-community/async-storage/jest/async-storage-mock';
+
+jest.mock('@react-native-community/async-storage', () => mockAsyncStorage);
+jest.mock('react-native/Libraries/Animated/src/NativeAnimatedHelper');
```

Comment:
a quoi ca sert ? ^^

Reply:
Ca permet de mocker asyncStorage et une librairie d'animation qui ne fonctionne pas bien en test. 
pour asynstorage: https://react-native-community.github.io/async-storage/docs/advanced/jest/
pour nativeAnimatedHelper: https://stackoverflow.com/questions/59587799/how-to-resolve-animated-usenativedriver-is-not-supported-because-the-native